packages/ml-kit/README.md

```diff
@@ -0,0 +1,156 @@
+|--------------------------------------------------|-------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <code>{classification&vert;regression}Aim</code> | `string`                                              | Objective of the classification/regression. Used to name the task.                                                                                                  |
+| `modelName`                                      | <code>string &vert; ModelNameResolver</code>          | Name of the model (without *tflite* extension) stored in the *ml-models* folder to use for this task, or a function that returns the name of the model when called. |
+| `tag` *(Optional)*                               | `string`                                              | Adds a tag to the name of the task to differentiate it from other tasks with other configurations.                                                                  |
+| `modelOptions` *(Optional)*                      | <code>ModelOptions &vert; ModelOptionsResolver</code> | Configuration to use with the model or a function that returns the configuration when called.                                                                       |
+
+- `ModelNameResolver: () => string`
+
+Useful to change the model used by a task at runtime. You can use the [`ModelManager`](#modelmanager) to obtain a list with the models that are available in the device.
+
+- `ModelOptions`
+
+| Property       | Type                                    | Description                                                                                                                                                                                                                                                                                               |
+|----------------|-----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `acceleration` | <code>DelegateType &vert; number</code> | Which type of acceleration to use when running the model. It can take the values `DelegateType.GPU` (GPU acceleration), `DelegateType.NNAPI` ([Android Neural Networks API](https://developer.android.com/ndk/guides/neuralnetworks) acceleration) or a number indicating the quantity of threads to use. |
+
+- `ModelNameResolver: () => ModelOptions`
```

Comment:
The bullets look a little bit strange to me. Maybe better to user headings?

Reply:
Yeah, I'm not convinced neither, I was following the format of other packages. I've restructured it a bit with sections...